File: KeyboardControl/VB/StartForm.vb
Imports System.ComponentModel
Imports System.Text
Imports DevComponents.DotNetBar

Namespace TestVirtualKeyboard
	Partial Public Class StartForm
		Inherits DevComponents.DotNetBar.Metro.MetroForm
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub metroTileItem1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles metroTileItem1.Click
			Dim form As New Form1()
			form.ShowDialog(Me)
		End Sub

		Private Sub metroTileItem2_Click(ByVal sender As Object, ByVal e As EventArgs) Handles metroTileItem2.Click
			Dim form As New Form2()
			form.ShowDialog(Me)
		End Sub

		Private Sub metroTileItem3_Click(ByVal sender As Object, ByVal e As EventArgs) Handles metroTileItem3.Click
			Dim form As New Form3()
			form.ShowDialog(Me)
		End Sub

		Private Sub metroTileItem4_Click(ByVal sender As Object, ByVal e As EventArgs) Handles metroTileItem4.Click
			Dim form As New Form4()
			form.ShowDialog(Me)
		End Sub

		Private Sub metroTileItem5_Click(ByVal sender As Object, ByVal e As EventArgs) Handles metroTileItem5.Click
			Dim form As New Form5()
			form.ShowDialog(Me)
		End Sub
	End Class
End Namespace

File: KeyboardControl/CS/StartForm.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevComponents.DotNetBar;

namespace TestVirtualKeyboard
{
    public partial class StartForm : DevComponents.DotNetBar.Metro.MetroForm
    {
        public StartForm()
        {
            InitializeComponent();
        }

        private void metroTileItem1_Click(object sender, EventArgs e)
        {
            Form1 form = new Form1();
            form.ShowDialog(this);
        }

        private void metroTileItem2_Click(object sender, EventArgs e)
        {
            Form2 form = new Form2();
            form.ShowDialog(this);
        }

        private void metroTileItem3_Click(object sender, EventArgs e)
        {
            Form3 form = new Form3();
            form.ShowDialog(this);
        }

        private void metroTileItem4_Click(object sender, EventArgs e)
        {
            Form4 form = new Form4();
            form.ShowDialog(this);
        }

        private void metroTileItem5_Click(object sender, EventArgs e)
        {
            Form5 form = new Form5();
            form.ShowDialog(this);
        }
    }
}